sales_team_security/models/res_partner.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    # add indexes for better performance on record rules
+    user_id = fields.Many2one(index=True)
+
+    def _remove_key_followers(self, partner):
+        for record in self.mapped("commercial_partner_id"):
+            # Look for delivery and invoice addresses
+            childrens = record.child_ids.filtered(
+                lambda x: x.type in {"invoice", "delivery"}
+            )
+            (childrens + record).message_unsubscribe(partner_ids=partner.ids)
+
+    def _add_followers_from_salesmans(self):
```

Comment:
Let's fix the incorrect grammar:

```suggestion
    def _add_followers_from_salesmen(self):
```